chapters/example.qmd

```diff
@@ -20,6 +20,12 @@ Note that there are five types of callouts, including:
 This is an example of a callout with a title.
 :::
 
+::: {.callout-warning}
+## Write a DMP
+
+It is important that you write a Data Management Plan. According to our university policy, everyone should do it!
```

Comment:
Shouldn't this information go somewhere else, for example in one of the sections instead of in a callout? Could you please look into this?